src/views/Farms/components/FarmTable/Liquidity.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react'
 import styled from 'styled-components'
 import { HelpIcon, Text, useTooltip } from '@pancakeswap-libs/uikit'
 import useI18n from 'hooks/useI18n'
+import BigNumber from 'bignumber.js'
 
 const ReferenceElement = styled.div`
   display: inline-block;
 `
 
 export interface LiquidityProps {
-  liquidity: number
```

Comment:
Why this is needed?

Reply:
It was a type mismatch, which TypeScript only caught after implementing the `useMemo` hook.

`farm.liquidity` gets assigned to this property, in `Farms.tsx` on line 269, and its type is `BigNumber`.

Reply:
Maybe we should open this as a seperate bugfix

Reply:
Created a separate PR [here](https://github.com/pancakeswap/pancake-frontend/pull/915)